homeassistant/components/universal/media_player.py

```diff
@@ -179,6 +183,7 @@ def __init__(
         self._state_template_result = None
         self._state_template = state_template
         self._device_class = device_class
+        self._unique_id = unique_id
```

Comment:
```suggestion
        self._attr_unique_id = unique_id
```

And therefore we can drop lines 307-311

Reply:
@dgomes  thanks! Addressed, mind taking another look/